tests/unit/test_projects.py

```diff
@@ -534,6 +534,34 @@ def test_slot_valid(self, slots, project_yaml_data):
         project = Project.unmarshal(project_yaml_data(slots=slots))
         assert project.slots == slots
 
+    def test_project_build_base_devel_grade_devel(self, project_yaml_data):
+        """When build_base is `devel`, the grade must be `devel`."""
+        project = Project.unmarshal(
+            project_yaml_data(build_base="devel", grade="devel")
+        )
+
+        assert project.grade == "devel"
+
+    def test_project_build_base_devel_grade_not_defined(self, project_yaml_data):
```

Comment:
Let's test this for a project whose build base is not devel

Reply:
Done!